.travis.yml

```diff
@@ -29,18 +29,19 @@ before_script:
     # Install the specified version of PHPUnit depending on the PHP version:
     if [[ -n "$TRAVIS_PHP_VERSION" ]]; then
       case "$TRAVIS_PHP_VERSION" in
-        7.2|7.1|7.0|nightly)
-          echo "Using PHPUnit 6.1"
-          composer global require "phpunit/phpunit=6.1.*"
+        7.3|nightly)
+          # when wp tests support phpunit 7 we'll update this.
+          echo "Using PHPUnit 6"
+          composer global require "phpunit/phpunit=6.5.*"
           ;;
-        5.6|5.5|5.4|5.3)
+        7.2|7.1|7.0)
+          echo "Using PHPUnit 6"
+          composer global require "phpunit/phpunit=6.5.*"
+          ;;
```

Comment:
is this right? I see it in other places, but gosh it sure done look wrong

Reply:
yes its bash/shell syntax